### PR TITLE
Important addition into the setup manual. 

### PR DESCRIPTION
**Pull request by swatisingh45**
### Description
Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change.

Fixes # [ISSUE]

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ Setup for developers (Windows)
     $ GRANT ALL PRIVILEGES ON systersdb TO <username created above>;
     ```
 1. Fill in the database details in `systers_portal/settings/dev.py`.
-1. Run `set SECRET_KEY=foobarbaz` in your terminal, ideally the secret key
+1. Run `set SECRET_KEY=foobarbaz` in your command terminal as well as in the psql terminal, ideally the secret key
```
[the remark this answers is not in the document]
Comment (swatisingh45) [reply README.md]: Hey! sorry for the late response, yes, as I was installing this on mac, the project wasn't running until the executed that command in both the shell; terminal as we well as psql session.

**Comment by sammy1997** [comment]
@swatisingh45 Are you working on this? CI build failing and 2 commits on this PR. Please resolve these issues

**Comment by sammy1997** [comment]
Also, link the issue in the PR description

**Comment by sammy1997** [comment]
Closing this due to no response from the contributor! cc: @abhi20161997 @sakshi1499

**Pull request closed without merging**